crates/fraud_proofs/global_merkle_root/storage/src/column.rs

```diff
@@ -61,31 +61,25 @@ impl TableColumn {
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
 pub enum Column {
     TableColumn(TableColumn),
-    MerkleMetadataColumn(TableColumn),
     MerkleDataColumn(TableColumn),
+    MerkleMetadataColumn,
 }
 
 impl Column {
     /// The total count of variants in the enum.
-    pub const COUNT: usize = TableColumn::COUNT + TableColumn::COUNT;
-
-    /// The start of the merkle metadata columns.
-    pub const MERKLE_METADATA_COLUMNS_START: u32 = u16::MAX as u32;
+    pub const COUNT: usize = TableColumn::COUNT + TableColumn::COUNT + 1;
```

Comment:
a comment about why the count has been incremented here would be nice :)

Reply:
Yeah I get why this looks confusing. This is actually two things:

1. The previous value was incorrect and should be `TableColumn::COUNT * 3` since we tripled the columns because each column had a data column and a metadata column. Now this PR changes so we reuse the metadata column for all columns, hence the last `TableColumn::COUNT` is replaced with `1`. We should add a comment about this, I'll do it in the base PR.